tests/browser.js

```diff
@@ -1,57 +1,33 @@
 "use strict";
 
-var phantom, phantomJsPrebuilt;
+var puppeteer; 
 try {
-  phantom = require("phantom");
-  phantomJsPrebuilt = require("phantomjs-prebuilt");
-} catch (err) {
-  throw new Error(
-    "Unable to run tests in PhantomJS because the required dependencies are " +
-    "not available. Please note that JSHint does not support development " +
-    "using versions of Node.js which are no longer maintained."
-  );
+  puppeteer = require('puppeteer');
+} catch (err) {	
+  throw new Error(	
+    "Unable to run tests in Chrome because the required dependencies are " +	
+    "not available. Please note that JSHint does not support development " +	
+    "using versions of Node.js which are no longer maintained."	
+  );	
```

Comment:
It's hard to see from GitHub.com, but this patch is adding trailing whitespace to some lines in this file. Could you remove the trailing whitespace?